mne/minimum_norm/tests/test_resolution_matrix.py

```diff
@@ -0,0 +1,107 @@
+"""
+Test the following properties of the resolution matrix.
+
+Resolution matrix is symmetrical for MNE (fixed and free source orientation).
+Resolution matrix has zero dipole localisation error for columns (PSFs).
+"""
+
```

Comment:
add license and author like here https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_event.py#L2